linera-views/src/common.rs

```diff
@@ -58,6 +58,16 @@ pub struct CommonStoreConfig {
     pub cache_size: usize,
 }
 
+impl Default for CommonStoreConfig {
```

Comment:
I'm not fond of this common default value, but I suppose we can remove it in the later refactoring.